lte/protos/session_manager.proto

```diff
@@ -737,90 +781,104 @@ enum SMSessionFSMState {
+    // LTESessionContextAccess lte_context = 1;
     M5GSMSessionContextAccess m5g_session_context_rsp = 2;
   //}
 }
 
-//AuthorizedQosRules
-//The purpose of the QoS rules information element is to indicate a set of QoS rules to be used by the UE,
-//where each QoS rule is a set of parameters.
-//a) for classification and marking of uplink user traffic; and
-//b) for identification of a QoS flow which the network is to use for a particular downlink user traffic.
-//As the UE needs to be aware of a QoS flow which the network is to use for a particular downlink user traffic
-//e.g. to determine whether a resource is available for downlink media of a media stream of an SDP media
-//description provided by the UE in an IMS session.
+// AuthorizedQosRules
+// The purpose of the QoS rules information element is to indicate a set of QoS rules to be used by the UE,
+// where each QoS rule is a set of parameters.
+// a) for classification and marking of uplink user traffic; and
+// b) for identification of a QoS flow which the network is to use for a particular downlink user traffic.
+// As the UE needs to be aware of a QoS flow which the network is to use for a particular downlink user traffic
+// e.g. to determine whether a resource is available for downlink media of a media stream of an SDP media
+// description provided by the UE in an IMS session.
+
+ enum AmbrUnit {
+   Kbps_0 = 0;
+   Kbps_1 = 1;
+   Kbps_4 = 2;
+   Kbps_16 = 3;
+   Kbps_64 = 4;
+ }
 
 message M5GSMSessionContextAccess{
-    //One byte, received from UE. For 5G pdu_session_id.
-    bytes pdu_session_id = 1;
-    //selected PDU session TS-24501 - 9.11.4.11
+    // One byte, received from UE. For 5G pdu_session_id. 
+    uint32 pdu_session_id = 1;
+    // selected PDU session TS-24501 - 9.11.4.11
     PduSessionType    pdu_session_type = 2;
-    //selected session and service continuty mode TS-24501 - 9.11.4.16
+    // selected session and service continuty mode TS-24501 - 9.11.4.16
     SscMode selected_ssc_mode = 3;
-    //TS-24501 - 9.11.4.3 - minimum 7 and max 65538, variable length
+    // TS-24501 - 9.11.4.3 - minimum 7 and max 65538, variable length
     repeated QosRules authorized_qos_rules = 4;
-    //TS-24501 - 9.11.4.14 PDU session Aggregate Maximum Bit Rate imported from subscriberdb.proto
-    AggregatedMaximumBitrate session_ambr = 5;
-    M5GSMCause M5gsm_cause = 6; //TS-24501 - 9.11.4.2
-    //PDU session is requested to be established as always-on PDU session.
-    bool always_on_pdu_session_indication = 7;
-    SscMode allowed_ssc_mode = 8; //Indicates SSC mode.
-    bool M5gsm_congetion_re_attempt_indicator = 9;
-    //IP address. Imported from session_manager.proto file.
-    RedirectServer pdu_address = 12;
+    // TS-24501 - 9.11.4.14 PDU session  Bit Rate imported from subscriberdb.proto
+    AmbrUnit downlink_unit_type = 5;
+    uint32   downlink_units = 6;   // Only to use lower 2 bytes (16 bit values)
+    AmbrUnit uplink_unit_type = 7;
```

Comment:
where is the AMBR information?

Reply:
enum AmbrUnit {
   Kbps_0 = 0;
   Kbps_1 = 1;
   Kbps_4 = 2;
   Kbps_16 = 3;
   Kbps_64 = 4;
 }    already defined.

Reply:
Described above